blazor/pivot-table/sorting.md

```diff
@@ -20,9 +20,9 @@ To have a quick glance on how to sort data in the Blazor Pivot Table, watch this
 
 Allows to order field members in rows and columns either in ascending or descending order. By default, field members in rows and columns are in ascending order.
 
-Member sorting can be enabled by setting the [EnableSorting](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.DataSourceSettingsModel-1.html#Syncfusion_Blazor_PivotView_DataSourceSettingsModel_1_EnableSorting) property in [PivotViewDataSourceSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotViewDataSourceSettings-1.html) class to **true**. After enabling this API, click the sort icon besides each field in row or column axis, available in the field list or grouping bar UI for re-arranging members either in ascending or descending order.
+Member sorting can be enabled by setting the [EnableSorting](https://help.syncfusion.com/cr/blazor#Syncfusion_Blazor_PivotView_DataSourceSettingsModel_1_EnableSorting/Syncfusion.Blazor.html) property in [PivotViewDataSourceSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotViewDataSourceSettings-1.html) class to **true**. After enabling this API, click the sort icon besides each field in row or column axis, available in the field list or grouping bar UI for re-arranging members either in ascending or descending order.
```

Comment:
Is this link loading properly?

Reply:
@berylouma, check the valid URL in below link, and update it.

<img width="980" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/44014283/4d9e4b5f-3ccf-44bb-ace6-6c4f096fa507">

https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotViewDataSourceSettings-1.html